### DISCUSSION TITLE
Custom production release from changes in /packages

### DISCUSSION THREAD

**Discussion by spanwair** [medusajs/medusa#2640] [RFC]
Hello, I got big plans with Medusajs and I know I will be touching already existing code in packages/medusa often. Is there a any way how to build those custom changes into new `server`. By now I am able to run development through `integration-tests/development/server.js` but do not know how to make it run in production mode. Any ideas

**Comment by adrien2p** [comment]
Is it to test it locally? In that case you can create a dummy project and i stall medusa from your local machine. Let me know what is your exact use case 👍
